src/main/scala/org/globalforestwatch/summarystats/forest_change_diagnostic/ForestChangeDiagnosticDataLossApproxYearlyTwoCategory.scala

```diff
@@ -0,0 +1,84 @@
+package org.globalforestwatch.summarystats.forest_change_diagnostic
```

Comment:
See comments below for the other TwoCategory one. Also, this seems like basically the same exact code as the other TwoCategory one but with ApproxYear. Is there any smart Scala way to like parametrize the type? This is getting to my limit of Scala knowledge though so don't have particular suggestions

Reply:
Yes, it would be nice to parameterize some of these.  I worked on this a bit today.  The problem is that to parameterize ForestChangeDiagnosticDataLossYearlyTwoCategory.scala to include Int or ApproxYear, you also have to parameterize ForestChangeDiagnosticDataLossYearlyCategory.scala and ForestChangeDiagnosticDataLossYearly.scala.  And FCDDataLossYearly.scala is hard, because you would need various (maybe implicit) methods to convert Int to ApproxYear in generic initialization methods.  The methods of the companion objects (like `fill` and `empty`) would also have to be parameterized.  Also, the circe encoding/decoding was very tricky to get right for these compound types, so I'm not sure I could get it all to work generically.

So, I think I'll also stay with this code for now, even though I know there's duplication.